Source/WEB_ROOT/Inventory/js/getAssetsAjax.js

```diff
@@ -0,0 +1,39 @@
+function MakeRequest(searchFor)

```

Comment:
This looks really clean John. Eventually you'll remove your debugging console logs and the code will be focused.